boa_engine/src/environments/compile.rs

```diff
@@ -288,7 +288,12 @@ impl Context {
     ///
     /// Panics if the global environment is not function scoped.
     #[inline]
-    pub(crate) fn create_mutable_binding(&mut self, name: Identifier, function_scope: bool) {
+    pub(crate) fn create_mutable_binding(
+        &mut self,
+        name: Identifier,
+        function_scope: bool,
+        deletable: bool,
```

Comment:
This seems to be used only for the `configurable` attribute. Wouldn't it make sense to call it `configurable`, so that it's more clear to understand what it means?

Reply:
Yep :)